From 1fb134dd57b8bf339ff3a67c7224c2f826b3deae Mon Sep 17 00:00:00 2001
From: Yu Jiahua <yujiahua1@huawei.com>
Date: Thu, 20 Jan 2022 12:22:37 +0800
Subject: [PATCH] Revert "sched: Access control for sysctl_update_load_latency"
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 1fb134dd57b8bf339ff3a67c7224c2f826b3deae


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4QU5Z?from=project-issue
CVE: NA

--------------------------------

This patch revert ias feature from open-euler kernel.

This reverts commit d668f5ea020df34b8753cd234d85746004d6f0df.

Signed-off-by: Yu Jiahua <Yujiahua1@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/sched/fair.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 6ad5d878223e..820c57fa460b 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -58,9 +58,6 @@ int sysctl_update_load_latency(struct ctl_table *table, int write,
 	int latency = sysctl_load_tracking_latency;
 	struct ctl_table t;
 
-	if (write && !capable(CAP_SYS_ADMIN))
-		return -EPERM;
-
 	t = *table;
 	t.data = &latency;
 	t.extra1 = &min;
-- 
2.26.2

